src/styles/partials/components/_table.scss

```diff
@@ -0,0 +1,88 @@
+/* BASIC TABLE STYLES */
+.table {
+  border: 1px solid $gray-light;
+  border-collapse: collapse;
+  color: #4c4c4c;
+  font-family: $default-heading-font;
+  font-size: 16px;
+  text-align: left;
+  width: 100%;
+  td {
+    border-bottom: 1px solid $gray-light;
+    padding: 15px;
+  }
+  thead {
+    th {
+      background-color: $white;
+      border-bottom: 5px solid $blue;
+      border-left: none;
+      color: $black;
+      font-family: $default-heading-font;
+      font-size: 18px;
+      font-weight: bold;
+      padding: 15px;
+    }
+  }
+  th {
+    border-bottom: 1px solid $gray-light;
+    border-left: 5px solid $blue;
+    color: $black;
+    padding-left: 20px;
```

Comment:
this should come from the spacing variables file

Reply:
Replaced with $padding-large. Will update in my next commit.